packer/plugin.go

```diff
@@ -200,6 +203,18 @@ func (c *PluginConfig) discoverExternalComponents(path string) error {
 		log.Printf("using external datasource %v", externallyUsed)
 	}
 
+	//Check for installed plugins using the `packer plugins install` command
+	pluginPaths, err = c.discoverSingle(filepath.Join(path, "*", "*", "*", fmt.Sprintf("packer-plugin-*%s", osArch)))
+	if err != nil {
+		return err
+	}
+
+	for pluginName, pluginPath := range pluginPaths {
+		if err := c.DiscoverMultiPlugin(pluginName, pluginPath); err != nil {
+			return err
+		}
+	}
+
 	pluginPaths, err = c.discoverSingle(filepath.Join(path, "packer-plugin-*"))
```

Comment:
Is it possible that these will overwrite the plugins installed with `plugins install`?

Reply:
You are correct. Manually installed plugins will still be the last plugins to be processed as the user may want to use a developer build of the plugin. This essentially keeps things as they are. Given that the the required_plugins block forces Packer to load the correct version of a plugin. We can safely say that a required_plugins block will always use the plugins installed with `packer init` or `packer plugins`. 

That said I think having a Debug log statement here indicating that a local development plugin for packer-plugin-happycloud was loaded here would be info to provide to the user.